destination.go

```diff
@@ -74,6 +76,10 @@ func (d *Destination) Write(ctx context.Context, recs []opencdc.Record) (int, er
 	b := &pgx.Batch{}
 	for _, rec := range recs {
 		var err error
+		rec, err = d.cleanRecord(rec)
```

Comment:
Left a suggestion about renaming this method below (name it `ensureStructuredRecord`)